pkgs/by-name/am/amneziawg-go/package.nix

```diff
@@ -0,0 +1,52 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+  testers,
+  amneziawg-go,
+  nix-update-script,
+}:
+
+buildGoModule rec {
+  pname = "amneziawg-go";
+  version = "0.2.12";
+
+  src = fetchFromGitHub {
+    owner = "amnezia-vpn";
+    repo = "amneziawg-go";
+    rev = "v${version}";
+    hash = "sha256-Xw2maGmNnx0+GO3OWS1Gu77oB9wh2dv+WobypQotUMA=";
+  };
+
+  postPatch = ''
+    # Skip formatting tests
+    rm -f format_test.go
+  '';
+
+  vendorHash = "sha256-zXd9PK3fpOx/YjCNs2auZWhbLUk2fO6tyLV5FxAH0us=";
+
+  subPackages = [ "." ];
+
+  ldflags = [
+    "-s"
+    "-w"
+  ];
+
+  strictDeps = true;
+
+  passthru = {
+    tests.version = testers.testVersion {
+      package = amneziawg-go;
+      version = "v${version}";
+    };
```

Comment:
That fails in ofborg

Reply:
Thanks, I should have checked that the tests worked before adding them.